Update app-templates.md to reflect retirement of certain app templates (#7875)
MicrosoftDocs/msteams-docs: msteams-platform/samples/app-templates.md

>
> * Microsoft is not actively managing the source code. The complete source code is open source and you can explore, fork, and modify the source code to meet your specific requirements.
> * You must not use Microsoft Power Platform to create apps that are to be published to the Teams app store. Microsoft Power Platform apps can be published to an organization’s app store only.

> > * Applications based on Poll, Survey, Checklist, Quiz and Training templates will be retired soon. The applications will stay active until February and users will be able to retrieve any data until early March. By the end of March the applications will be fully retired.

Reviewer: ```suggestion
> * Applications based on Poll, Survey, Checklist, Quiz, and Training templates will be retired completely by March 31, 2023. The applications will stay active until February 28, 2023 and you will be able to retrieve any data until early March. 
```

| Celebrations |Celebrations is a Teams app that helps team members to celebrate each others' birthdays, anniversaries, and other recurring events. It remembers special occasions of all the team members and sends a friendly message in all the teams selected at the time of event creation, to make the team members feel special on their day. The app provides an easy interface for all the team members to personally add and view their events and also allows the user to select the teams in which the events get shared. | [Celebrations](https://github.com/OfficeDev/microsoft-teams-celebrations-app) |

> | Checklist | Note: The Checklist App Template will be retired soon. For more details refer the note section [here]([url](https://learn.microsoft.com/en-us/microsoftteams/platform/samples/app-templates#app-template-code-samples)). <br/> Checklist is a custom Teams [message extension](../messaging-extensions/what-are-messaging-extensions.md) app that enables you to collaborate with your team by creating a shared checklist in a chat or channel. The app is supported across all Teams platform clients, such as desktop browser, iOS, and Android. The app is ready for deployment as part of your Microsoft 365 subscription.  | [Checklist](https://github.com/OfficeDev/microsoft-teams-checklist-app)|

Author: ```suggestion
| Checklist | Note: The Checklist App Template will be retired soon. For more details refer the note section [here]([url](../samples/app-templates#app-template-code-samples)). <br/> Checklist is a custom Teams [message extension](../messaging-extensions/what-are-messaging-extensions.md) app that enables you to collaborate with your team by creating a shared checklist in a chat or channel. The app is supported across all Teams platform clients, such as desktop browser, iOS, and Android. The app is ready for deployment as part of your Microsoft 365 subscription.  | [Checklist](https://github.com/OfficeDev/microsoft-teams-checklist-app)|
```

| Poll| Note: The Poll App Template will be retired soon. For more details refer the note section [here]([url](https://learn.microsoft.com/en-us/microsoftteams/platform/samples/app-templates#app-template-code-samples)). <br/> Poll is a custom Teams [message extension](../messaging-extensions/what-are-messaging-extensions.md) app that enables you to quickly create and send polls in a chat or a channel to gather team opinions and preferences. The app is supported across all Teams platform clients, such as desktop, browser, iOS, and Android and is ready for deployment as part of your Microsoft 365 subscription.|[Poll](https://github.com/OfficeDev/microsoft-teams-poll-app) |
| Quick Responses| Quick Responses is a Teams app that delivers a robust solution for effectively answering users' commonly asked questions FAQs. Instead of answering each query manually and continuously repeating information, the app builds a library of responses for an interactive user experience through Teams [message extensions](../messaging-extensions/what-are-messaging-extensions.md).|[Quick Responses](https://github.com/OfficeDev/microsoft-teams-apps-quickresponses) |

> |Quiz | Note: The Quiz App Template will be retired soon. For more details refer the note section [here]([url](https://learn.microsoft.com/en-us/microsoftteams/platform/samples/app-templates#app-template-code-samples)). <br/> Quiz is a custom [Teams message extension](../messaging-extensions/what-are-messaging-extensions.md) app that enables you to create a quiz within a chat or a channel for knowledge check and instantaneous results. You can use Quiz for, In-class and offline exams, Knowledge check within team, and for fun quizzes within a team. Quiz app is supported across multiple platforms, such as Teams desktop, browser, iOS, and Android clients. This app is ready for deployment as part of your existing Microsoft 365 subscription.|[Quiz](https://github.com/OfficeDev/microsoft-teams-apps-quiz) |

Author: ```suggestion
|Quiz | Note: The Quiz App Template will be retired soon. For more details refer the note section [here]([url](../samples/app-templates#app-template-code-samples)). <br/> Quiz is a custom [Teams message extension](../messaging-extensions/what-are-messaging-extensions.md) app that enables you to create a quiz within a chat or a channel for knowledge check and instantaneous results. You can use Quiz for, In-class and offline exams, Knowledge check within team, and for fun quizzes within a team. Quiz app is supported across multiple platforms, such as Teams desktop, browser, iOS, and Android clients. This app is ready for deployment as part of your existing Microsoft 365 subscription.|[Quiz](https://github.com/OfficeDev/microsoft-teams-apps-quiz) |
```

|Time Tally |A project can include multiple tasks, and various projects can be assigned to employees. Managers are required to understand the project progress through the time spent by the employees on these tasks. This can be a cumbersome activity, as the employees need to fill in the timesheets. Time Tally app enables employees to fill their timesheets quickly, using the mobile device, and managers don't have to follow up with employees on the timesheet entry. Managers get to view the project utilization based on resources, and they can approve or reject the entries. Reminder notifications are sent to ensure timesheet compliance. Also, historical data and utilizations are available for analytics. |[Time Tally](https://github.com/OfficeDev/microsoft-teams-apps-timetally) |

> | Training | Note: The Training App Template will be retired soon. For more details refer the note section [here]([url](https://learn.microsoft.com/en-us/microsoftteams/platform/samples/app-templates#app-template-code-samples)). <br/> Training is a custom [Teams message extension](../messaging-extensions/what-are-messaging-extensions.md) app that enables users to publish a training within a chat or a channel for offline knowledge sharing and upskilling. The app is supported across multiple Teams platform clients, such as desktop, browser, iOS, and Android. This app is ready for deployment as part of your Microsoft 365 subscription.|[Training](https://github.com/OfficeDev/microsoft-teams-apps-training) |

Author: ```suggestion
| Training | Note: The Training App Template will be retired soon. For more details refer the note section [here]([url](../samples/app-templates#app-template-code-samples)). <br/> Training is a custom [Teams message extension](../messaging-extensions/what-are-messaging-extensions.md) app that enables users to publish a training within a chat or a channel for offline knowledge sharing and upskilling. The app is supported across multiple Teams platform clients, such as desktop, browser, iOS, and Android. This app is ready for deployment as part of your Microsoft 365 subscription.|[Training](https://github.com/OfficeDev/microsoft-teams-apps-training) |
```
